modules/kubernetes-addons/main.tf

```diff
@@ -179,14 +179,18 @@ module "crossplane" {
 }
 
 module "external_dns" {
-  count             = var.enable_external_dns ? 1 : 0
-  source            = "./external-dns"
+  source = "./external-dns"
+
+  count = var.enable_external_dns ? 1 : 0
+
```

Comment:
Nit question: does it matter if source comes before count (and do we need extra line in-between)? looking at other modules it's usually source->count

Reply:
does it matter - no. its just my OCD in that its typically source directly under the module declaration. the other changes are just whitespace to break out to sections:
```hcl
module "foo" {
  source = "<URL/PATH>

  count or for_each
  
  parameters
  
  tags
  
  depends_on
}
```